### DISCUSSION TITLE
1.24.1 + Wear OS 2.15.1

### DISCUSSION THREAD

**Discussion by jamie-mh** [stratumauth/app#1104] [General]
- **General**: Fixed issue with clipped text when using larger fonts
- **General**: Improvements to 2FAS import
- **General**: 16 new icons and improvements to existing icons
- **General**: Translation updates

<hr /><em>This discussion was created from the release <a href='https://github.com/jamie-mh/AuthenticatorPro/releases/tag/1.24.1'>1.24.1 + Wear OS 2.15.1</a>.</em>

**Comment by pizofreude** [comment]
I hope you guys take over the Desktop app market as well. Would be great in there's import feature from Twilio as well. Cheers.
